src/smcntrpmf.adoc

```diff
@@ -16,10 +16,10 @@ Smcntrpmf remedies these issues by introducing privilege mode filtering for the
 
 mcyclecfg and minstretcfg are 64-bit registers that configure privilege mode filtering for the cycle and instret counters, respectively.  
 
-[cols="^1,^1,^1,^1,^1,^1,^5",stripes=even,options="header"]
+[cols="^1,^1,^1,^1,^1,^1,^5,^1",stripes=even,options="header"]
 |====
-|63 |62 |61 |60 |59 |58 |57:0
-|0 |MINH |SINH |UINH |VSINH |VUINH |_WPRI_ 
+|63 |62 |61 |60 |59 |58 |57:0 |56
```

Comment:
This should be:
```
|63 |62      |61      |60      |59        |58         |57:0 
|0  |MINH    |SINH    |UINH    |VSINH     |VUINH      |_WPRI_ 
```

Reply:
So there should be no bit 56?  That contradicts Greg's comment to add them both back into the chapters.

Reply:
Re Bill last post ... Yes, there should be bits 57 and 56 - which are both WPRI and described as Reserved.  The "57:0" should just be corrected to "57" - which still leaves bit 56 as is in the PR.

Reply:
And it's a separate matter (per my post two minutes ago, whether these two specs say something explicitly about bit [55:0] or not.

Reply:
The PR has been updated to fix 57:0 and leaves 56, as per @gfavor.

Reply:
Wow, lots of comments rolled in while I was reviewing, so some duplication here.  First, following 57:0 with 56 (as shown above currently) doesn't make sense.  You could have `57 | 56 | 55:0`, but they'd all be WPRI, so I don't see the advantage to that.  

@gfavor, what you're proposing changes the ratified Smcntrpmf ISA.  These *cfg registers are not analogs for mhpmeventX, they only replicate the upper bits.  My understanding for this PR is only that 57:56 should be reserved in both Smcntrpmf and Sscofpmf, so let's stick with that.  Which requires no changes to Smcntrpmf.  If there are other changes to be considered, that should be a separate discussion.  But as stated in my other comments, I don't think any other change to Smcntrpmf makes sense.

Reply:
Per the ARC's final resolution, bits [57:56] for Sscofpmf are to be Reserved and WPRI.  Wasn't Smcntrpmf intending to match whatever Sscofpmf says for the high 8 bits that Sscofpmf spec'ed?

Or is Smcntrpmf specifically not striving to fully match what Sscofpmf says for the high bits?